modules/ROOT/pages/_partials/cookbook-dw/parse-dates-ex1/transform.dwl

```diff
@@ -0,0 +1,7 @@
+%dw 2.0
+output application/xml
+fun normalize(date) = (date) replace "/" with "-" replace "." with "-"
+---
+dates: (payload.dates mapObject {
+    normalized_as_string: normalize($)
```

Comment:
```suggestion
    normalized_as_string: normalize($.date)
```

Reply:
I noticed this under https://docs.mulesoft.com/mule-runtime/4.3/dataweave-cookbook-parse-dates, unless it has changed

Reply:
there was an error on that page in the doc site.